src/run/mod.rs

```diff
@@ -38,6 +38,29 @@ fn show_banner() {
     debug!("codspeed v{}", VERSION);
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, ValueEnum, Default)]
+pub enum UnwindingMode {
+    /// Use the frame pointer for unwinding. Requires the binary to be compiled with frame pointers enabled.
+    #[clap(name = "fp")]
+    FramePointer,
+
+    /// Use DWARF unwinding. This does not require any special compilation flags and is enabled by default.
+    #[default]
+    Dwarf,
+}
+
+#[derive(Args, Debug, Clone)]
+pub struct PerfRunArgs {
+    /// Enable the performance runner, which uses `perf` to collect performance data.
+    /// This is only supported on Linux.
+    #[arg(long, env = "CODSPEED_PERF_ENABLED", default_value_t = false)]
+    enable_perf: bool,
+
+    /// The unwinding mode that should be used with perf to collect the call stack.
+    #[arg(long, env = "CODSPEED_PERF_UNWINDING_MODE")]
+    unwinding_mode: Option<UnwindingMode>,
```

Comment:
from your comment then, making this `--perf-unwinding-mode` would make more sense so you can either just rename the CLI flag or the variable !